src/commands/ipmi.rb

```diff
@@ -51,28 +51,28 @@ def run_vm(node)
       end
 
       def run_baremetal(node)
-        puts "#{node.name}: #{SystemCommand.run(ipmi_command(node.name))}"
+        puts "#{node.name}: #{SystemCommand.run(ipmi_command(node))}"
       end
 
-      def ipmi_command(node_name)
-        create_ipmitool_command(
-          host: "#{node_name}.bmc",
-          arguments: command_argument
-        )
+      def ipmi_command(node)
+        opt = { host: "#{node.name}.bmc", arguments: command_argument }
+        create_ipmitool_command(node, **opt)
       end
 
-      def create_ipmitool_command(host:, arguments:)
-        "ipmitool -H #{host} -I lanplus #{render_credentials} #{arguments}"
+      def create_ipmitool_command(node, host:, arguments:)
+        <<~ECMD.squish
```

Comment:
What does `ECMD` stand for?

Reply:
End command. `rubocop` complains if you use `EOF`

Reply:
That seems worse as it's non-standard but no more descriptive (`EOF` is a standard delimiter for heredocs) - I think you can ignore rubocop in this case; I'll push a change.